pkgs/development/python-modules/expandvars/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "expandvars";
+  version = "0.9.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-ag54IeVf8atE+MoItQ0rLQHAdqm8l52pQEZ8t/EFxWU=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+  ];
```

Comment:
Since this is only a build-time dependency, can we make this:
```
nativeBuildInputs = [
  setuptools
  wheel
];
```
I added `wheel` as a future-proofing measure (since I'm working on making that explicit, since not all builds require it).

Maybe it's worth submitting a PR upstream to change its build system in pyproject.toml to poetry-core, since the rest of it uses poetry too.

Reply:
Done!